configuration.go

```diff
@@ -0,0 +1,156 @@
+package raft
+
+import "fmt"
+
+// ServerSuffrage determines whether a Server in a Configuration gets a vote.
+type ServerSuffrage int
+
+const (
+	// Voter is a server whose vote is counted in elections and whose match index
+	// is used in advancing the leader's commit index.
+	Voter ServerSuffrage = 1
```

Comment:
For sure - we use `iota` in log.go for the actual log message types so we are already using it for critical numbering / add only at the end kinds of things.